asgiref/py.typed

```diff
@@ -0,0 +1 @@
+Marker
```

Comment:
```suggestion
```

Reply:
This used not to work, although it does now...